ccd-definition/CaseEventToComplexTypes/Languages-hnl-legal-rep.json

```diff
@@ -133,62 +87,16 @@
     "CaseFieldID": "applicant2DQLanguage",
     "ListElementCode": "court",
     "FieldDisplayOrder": 2,
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
+    "Publish": "Y"
   },
   {
     "ID": "GAUrgencyRecordPage",
     "CaseEventID": "CLAIMANT_RESPONSE",
     "CaseFieldID": "applicant2DQLanguage",
     "ListElementCode": "documents",
     "FieldDisplayOrder": 3,
-    "DisplayContext": "MANDATORY"
-  },
-  {
-    "ID": "Languages",
```

Comment:
these fields are just moved to ccd-definition/CaseEventToComplexTypes/LanguagesLRspec-hnl-legal-rep.json because they the caseeventid is DEFENDANT_RESPONSE_SPEC or CLAIMANT_RESPONSE_SPEC